docs/rail/output.md

```diff
@@ -9,7 +9,7 @@ The `<output>...</output>` element of a `RAIL` spec is used to give precise spec
 
 Example:
 
-=== "RAIL Spec"
+=== "JSON RAIL Spec"
```

Comment:
nit: Here, can you add a TODO to add formatting so that there's nesting between the spec and outputs for each output type? Right now, all 4 things (2 specs and 2 outputs) are flat. Ideally there should be some nesting like:


------------------------------------------
| JSON Outputs         | Str outputs.             |
------------------------------------------
| RAIL Spec | Output | RAIL Spec | Output |